css/wishlist.css

```diff
@@ -0,0 +1,85 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+main {
+  margin: var(--size-8) auto;
+}
+
+main .h1 {
+  text-align: center;
+}
+
+.wishlist {
+  display: grid;
+  margin-top: var(--size-4);
+  grid-template-columns: auto;
+  padding: var(--size-4);
+  grid-column-gap: 10px;
+  grid-row-gap: 50px;
+  justify-content: center;
+}
+
+.logo {
+  margin-right: var(--size-2);
+}
+
+.btn_logout {
+  padding: var(--size-3) var(--size-6);
+}
+
+@media only screen and (min-width: 30rem) {
```

Comment:
Its a good practice to place media queries at the end of CSS file

Reply:
Thanks for pointing that out. Done!